### PR TITLE
Add action for Hyp3 Deployment for NISAR GUNW products

### PR DESCRIPTION
**Pull request by cmarshak**
Copy the entry from the 1 file below and then modified:

```
 - environment: hyp3-a19-jpl
            domain: hyp3-a19-jpl.asf.alaska.edu
            template_bucket: cf-templates-v4pvone059de-us-west-2
            image_tag: latest
            product_lifetime_in_days: 180
            quota: 0
            job_files: job_spec/INSAR_ISCE.yml job_spec/INSAR_ISCE_TEST.yml
            instance_types: c5d.xlarge
            default_max_vcpus: 1600
            expanded_max_vcpus: 1600
            required_surplus: 0
            security_environment: JPL-public
            ami_id: /aws/service/ecs/optimized-ami/amazon-linux-2/recommended/image_id
            distribution_url: ''
```

Changing `hyp3-a19-jpl` to `hyp3-nisar-jpl` and updating the template bucket use cloud formation to generate a bucket for storing cloud formation templates.

### CODE REVIEW

File: .github/workflows/deploy-enterprise.yml [hunk 1]
```diff
@@ -42,6 +42,21 @@ jobs:
...
+            template_bucket: cf-templates-gdeyr9hh8rzs-us-west-2
+            image_tag: latest
+            product_lifetime_in_days: 180
+            quota: 0
+            job_files: job_spec/INSAR_ISCE.yml job_spec/INSAR_ISCE_TEST.yml
+            instance_types: c5d.xlarge
+            default_max_vcpus: 1600
+            expanded_max_vcpus: 1600
+            required_surplus: 0
+            security_environment: JPL-public
```
Comment (jhkennedy) [inline .github/workflows/deploy-enterprise.yml:56]:
```suggestion
            security_environment: JPL
```
I forgot to mention this bit -- we need to use the `JPL` security environment until the public bucket request is done. After we can switch it to `JPL-public` which will make the HyP3 content bucket public.

**Review by jhkennedy — CHANGES_REQUESTED** [review]
Do you also want to do `hyp3-tibet-jpl` in this same PR, or separate?

**Comment by cmarshak** [comment]
> Do you also want to do `hyp3-tibet-jpl` in this same PR, or separate?

I am going to create a new pull request.

**Comment by jhkennedy** [comment]
Looks good to me! Here's our (ASF) todos before merging/deploying:
- [ ] create an `hyp3-nisar-jpl` GitHub Actions environment like the `hyp3-a19-jpl` one
- [ ] populate the environment with necessary secrets
    - [ ] `AWS_REGION`
    - [ ] `V2_AWS_ACCESS_KEY_ID`
    - [ ] `V2_AWS_SECRET_ACCESS_KEY`
    - [ ] `EDL_USERNAME`
    - [ ] `EDL_PASSWORD`
    - [ ] `VPC_ID`
    - [ ] `SUBNET_ID`
    - [ ] CERTIFICATE_ARN`
- [ ] verify service user access keys (stored in account secrets manager) work

@cmarshak do you want to use the same Earthdata Login user as `hyp3-a19-jpl`, or create a new `hyp3-nisar` user?

**Comment by cmarshak** [comment]
> @cmarshak do you want to use the same Earthdata Login user as `hyp3-a19-jpl`, or create a new `hyp3-nisar` user?

Let's use the same user for `a19` and elsewhere - will be too complicated to create multiple "operational users" that everyone has access to.

**Comment by jhkennedy** [comment]
I've created the GitHub Actions environment and have populated all the secrets but `CERTIFICATE_ARN`. A new TLS certificate needs to be generated by ASF's Platform Team for `hyp3-nisar-jpl.asf.alaska.edu`, which I've requested. Then I will need to uploaded to the AWS account before I know it's ARN.

**Review by jhkennedy — APPROVED** [review]
TLS certificate was uploaded to the Certificate Manager and the the GitHub Actions environment should be fully populated! :shipit:

**Pull request merged**